web/docs/auth/social-auth/discord.md

```diff
@@ -458,6 +461,12 @@ export function getConfig() {
 
 <UsingAuthNote />
 
+When you receive the `user` object [on the client or the server](../overview.md#accessing-the-logged-in-user), you'll be able to access the user's Discord ID like this:
```

Comment:
Every social provider had the same docs structure, this was missing, so I added it now.